DependencyInjection/FOSRestExtension.php

```diff
@@ -326,11 +326,20 @@ private function loadException(array $config, XmlFileLoader $loader, ContainerBu
                 ->replaceArgument(0, $config['exception']['codes']);
             $container->getDefinition('fos_rest.exception.messages_map')
                 ->replaceArgument(0, $config['exception']['messages']);
+
+            if (isset($config['exception']['debug'])) {
+                $container->getDefinition('fos_rest.exception.controller')
+                    ->replaceArgument(2, $config['exception']['debug']);
+                $container->getDefinition('fos_rest.serializer.exception_normalizer.jms')
+                    ->replaceArgument(1, $config['exception']['debug']);
+                $container->getDefinition('fos_rest.serializer.exception_normalizer.symfony')
+                    ->replaceArgument(1, $config['exception']['debug']);
+            }
```

Comment:
I would not wrap this in an `if` statement. Let's just always do the replacement here.